maxim/tests/test_prompts.py

```
@@ -45,47 +38,66 @@ def tearDown(self):
+        self.assertEqual(prompt.provider, "openai")
+        self.assertEqual(prompt.messages[0].content, "You are a helpful assistant")
+        self.assertEqual(len(prompt.messages), 1)
+        try:
+            resp = prompt.run(
+                "What is Cosmos about?",
+            )
+        except Exception as e:
+            self.fail(f"prompt.run() raised an exception: {e}")
 
-        print(f"Provider: {prompt.provider}")
+    def test_getPrompt_with_multiselect_deployment_variables_and_execute(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_getPrompt_with_multiselect_deployment_variables_and_execute`

Add return type annotation: `None`

(ANN201)